src/__test__/actions/assetTypes.actions.test.js

```diff
@@ -1,43 +1,82 @@
 // third-party libraries
 import expect from 'expect';
-import axios from 'axios';
-import MockAdapter from 'axios-mock-adapter';
 import thunk from 'redux-thunk';
 import configureMockStore from 'redux-mock-store';
+import moxios from 'moxios';
```

Comment:
@HawiCaesar why was this installed? Isn't `axios-mock-adapter` doing a similar job?

Reply:
@sachungo I installed it but it was actually going to the url. Moxios does a better of job of testing and testing failure scenarios.

Reply:
noted